restaurant/pkg/data/queries.go

```diff
@@ -0,0 +1,7 @@
+package data
+
+const readAllOrdersQuery = "SELECT id, date, table_number, waiters.full_name, price, payment from orders join waiters on waiters.waiter_id = orders.waiter_id"
+const readOrdersQuery = "SELECT id, date, table_number, waiters.full_name, price, payment from orders join waiters on waiters.waiter_id = orders.waiter_id WHERE orders.id = $1"
```

Comment:
`SELECT` was written by uppercase, `FROM` was written by lowercase...
reread code convention for sql queries and rewrite queries